src/Event/EventApplication/Events/List.cs

```diff
@@ -0,0 +1,26 @@
+﻿using DAL.Events;
+using DAL.Events.Repository;
+using MediatR;
+
+namespace EventApplication.Events
+{
+	public class List
+	{
+		public class Query : IRequest<IList<Event>> { }
+
+		public class RequestHandler : IRequestHandler<Query, IList<Event>>
```

Comment:
This is a strange implementation from my perspective.
1 You need to create additional file `List` on each request just to resolve this request - this is strange
2 The name of the file itself is confusing by intersecting with Collections reserved word
3 You keep 3 classes in fact in one file - request, handler and some kind of wrapper
Instead I would recommend having something like `GetAllEventsQuery` and `GetAllEventsQueryHandler` in separate files

Reply:
I just liked how it looked from controller side to use `new Events.List.Query` syntax and less files in repository to go between them also felt nice.